.github/workflows/code-formatting.yml

```diff
@@ -7,6 +7,21 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
     branches: [master]
 
 jobs:
+  isort:
+    name: Check valid import formatting with isort
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install isort
+        run: pip install isort==5.6.4
+      - name: Run isort
+        run: isort --settings-path=./pyproject.toml --check-only .
```

Comment:
Not a strong opinion, but how about adding `--diff` option.
```
 -df, --diff           Prints a diff of all the changes isort would make to a
                        file, instead of changing it in place
```

Reply:
Hey @nathanpainchaud, `--diff` would be nice to see actually.

Reply:
The `--check-only` option already lists the file that would be changed, without changing them in-place. From what I understand, `--diff` would print all the changes, even inside each file. I've got no problem adding it, I just want to make sure it wouldn't be considered too noisy in the end.